pom.xml

```diff
@@ -42,30 +42,378 @@
     <jackson.version>1.9.13</jackson.version>
     <json-simple.version>1.1.1</json-simple.version>
+    <geronimo-servlet_2.5_spec.version>1.2</geronimo-servlet_2.5_spec.version>
+ 	<com.sun.tools.version>1.6.0</com.sun.tools.version>
+ 	<jersey.version>1.9</jersey.version>
+ 	<rest-assured.version>2.4.0</rest-assured.version>
+ 	<commons-lang3.version>3.3.2</commons-lang3.version>
     <junit.version>4.11</junit.version>
     <mockito.version>1.9.5</mockito.version>
     <requirejs.version>2.1.14</requirejs.version>
     <requirejs.version.full>${requirejs.version}-1</requirejs.version.full>
+    <angularjs.version>1.2.21</angularjs.version>
+    <jquery.version>1.9.1</jquery.version>
     <slf4j.version>1.7.3</slf4j.version>
 
     <!-- Other pentaho dependency versions -->
-    <kettle.version>6.0-SNAPSHOT</kettle.version>
-    <notification.version>6.0-SNAPSHOT</notification.version>
+    <kettle.version>${project.version}</kettle.version>
+    <notification.version>${project.version}</notification.version>
+    <pentaho-i18n-bundle.version>${project.version}</pentaho-i18n-bundle.version>
     <bean-matchers.version>0.9</bean-matchers.version>
+ 	
+    <source.jdk.version>1.6</source.jdk.version>
+    <target.jdk.version>1.6</target.jdk.version>
+ 	
+ 	<plugin.org.apache.maven.plugins.maven-resources-plugin.version>2.7</plugin.org.apache.maven.plugins.maven-resources-plugin.version>
+    <plugin.org.apache.maven.plugins.maven-compiler-plugin.version>3.0</plugin.org.apache.maven.plugins.maven-compiler-plugin.version> 
+    <plugin.org.apache.maven.plugins.maven-javadoc-plugin.version>2.10.2</plugin.org.apache.maven.plugins.maven-javadoc-plugin.version>
+    <plugin.org.apache.maven.plugins.maven-jar-plugin.version>2.5</plugin.org.apache.maven.plugins.maven-jar-plugin.version>
+    <plugin.org.apache.felix.maven-bundle-plugin.version>2.3.7</plugin.org.apache.felix.maven-bundle-plugin.version>
+    <plugin.org.codehaus.mojo.build-helper-maven-plugin.version>1.5</plugin.org.codehaus.mojo.build-helper-maven-plugin.version>
+    <plugin.org.apache.karaf.tooling.karaf.karaf-maven-plugin.version>${karaf.version}</plugin.org.apache.karaf.tooling.karaf.karaf-maven-plugin.version>
+	<plugin.com.github.klieber.phantomjs-maven-plugin.version>0.4</plugin.com.github.klieber.phantomjs-maven-plugin.version>
+    <plugin.com.github.searls.jasmine-maven-plugin.version>1.3.1.5</plugin.com.github.searls.jasmine-maven-plugin.version>
+    <plugin.com.github.timurstrekalov.saga-maven-plugin.version>1.5.3</plugin.com.github.timurstrekalov.saga-maven-plugin.version>
   </properties>
 
-  <scm>
-    <connection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</connection>
-    <url>scm:git:git@github.com:${github.user}/${project.artifactId}.git</url>
-    <developerConnection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</developerConnection>
-    <tag>HEAD</tag>
-  </scm>
+  
+  <!-- copied from documentation pom.xml -->
```

Comment:
It sets the value for the toolsjar property. This property is being used in projects parent pom in the dependency com.sun.tools. For me it makes sense, because as far as I understood, the profiles purpose is dependent on the OS, so later other child POMs might use it.